packages/ui-components/src/components/range/range.tsx

```diff
@@ -36,22 +41,22 @@ export class KvRange implements IRange, IRangeEvents {
 	private applyCssStyles = () => {
 		const rangeInputValue = this.getRangeElement();
 
-		const selector = this.el.shadowRoot.getElementById('select-value');
-
 		const inputValue = this.getInputValue(rangeInputValue);
 		const percentage = getInputPercentageFromValue(inputValue, this.min, this.max);
 		const inputOffSet = getInputOffset(percentage);
 		const valueOffset = getValueOffset(percentage, inputValue);
```

Comment:
I believe this valueOffset is only used inside the !this.hideLabel if. If so, maybe would be better to move this calculation inside the if